browser-test/src/support/admin_translations.ts

```diff
@@ -9,6 +9,19 @@ type ProgramStatusTranslationParams = {
   statusEmail?: string
 }
 
+type ProgramTranslationParams = {
+  name: string
+  description: string
+  blockName: string
+  blockDescription: string
+  statuses: ProgramStatusTranslationParams[]
+  shortDescription?: string
```

Comment:
apologies for not noticing this before! now that we're fully on northstar, short description should be required and "description" should be optional. i think this changes the logic below as well, but i'll comment there.